include/openssl/evp.h

```diff
@@ -141,6 +140,12 @@ OPENSSL_EXPORT int EVP_PKEY_id(const EVP_PKEY *pkey);
 // otherwise.
 OPENSSL_EXPORT int EVP_PKEY_type(int nid);
 
+// EVP_MD_get0_name returns the short name of |md|
+OPENSSL_EXPORT const char *EVP_MD_get0_name(const EVP_MD *md);
+
+// EVP_MD_name calls |EVP_MD_get0_name|
+OPENSSL_EXPORT const char *EVP_MD_name(const EVP_MD *md);
+#define EVP_MD_name EVP_MD_name
```

Comment:
We should move these down to the section below or the prefix build could fail.
https://github.com/aws/aws-lc/blob/95cc2ce94a00304eaa887a80b851b89e5f0c65d8/include/openssl/evp.h#L1164-L1169